scripts/generate_llms.py

```diff
@@ -0,0 +1,123 @@
+import os
+import re
+
+# Set the base directory to the root of wormhole-docs
+base_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..'))
+docs_dir = os.path.join(base_dir, 'wormhole-docs')
+output_file = os.path.join(docs_dir, 'llms.txt')
+snippet_dir = os.path.join(docs_dir, '.snippets')
+
+# Regex to find lines like: --8<-- 'code/build/applications/...'
+SNIPPET_REGEX = r"--8<--\s*['\"]([^'\"]+)['\"]"
+
+
+def get_all_markdown_files(directory):
+    """
+    Recursively collect all markdown (.md, .mdx) files from subdirectories of the given directory,
+    skipping files directly in the root of "wormhole-docs".
+    """
+    results = []
+    if not os.path.exists(directory):
+        print(f"Docs directory not found: {directory}")
+        return results
+
+    for root, _, files in os.walk(directory):
+        # Skip the root directory
+        if root == directory:
+            continue
+
+        for file in files:
+            if file.endswith(('.md', '.mdx')):
+                results.append(os.path.join(root, file))
+
+    return results
+
+
+def build_index_section(files):
+    section = "# List of doc pages (from learn/):\n"
```

Comment:
```suggestion
    section = "# List of doc pages:\n"
```

I was trying out things only in the learn section and forgot to delete this - the code here takes all sections